app/components/job-list-item/job-list-item.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="job-list-container table clickable" (click)="onListItemClick()">
+  <div class="table-row">
+    <div class="table-cell min-width-column company-logo-container">
+      <img  class="company-logo no-select" [src]="(job.company.companyLogo && job.company.companyLogo.imageUrlSmall) || 'assets/images/placeholder-logo.png'">
+    </div>
+    <div class="table-cell">
+      <div class="table job-description-container">
+        <div class="table-row">
+          <div class="table-cell">
+            <div class="job-name">{{job.name}}</div>
+            <div class="job-description">{{job.shortDescription}}</div>
+            <div class="job-address">
+              <i class="icon-ic-location"></i>
+              {{job.street}} {{job.zip}}
+            </div>
+          </div>
+          <div class="table-cell job-rate text-center">
+            <div>
+              <div class="total-salary" *ngIf="isCompanyUser">
+                {{job.hourlyPay.rateExcludingVAT * job.hours}} {{job.hourlyPay.currency}}
+              </div>
+              <div class="total-salary" *ngIf="!isCompanyUser">{{job.hourlyPay.grossSalary * job.hours}} {{job.hourlyPay.currency}}</div>
```

Comment:
I've added a key to the job response `amount` that you can use, so you don't have to do this calculation client side.